client/scss/components/forms/_field-comment-control.scss

```diff
@@ -53,14 +53,6 @@ $button-padding: theme('spacing.2');
     transform: translateY(0);
     top: 0;
   }
-
-  .w-field--date_field &,
-  .w-field--date_time_field &,
-  .w-field--time_field & {
-    position: relative;
-    transform: translateY(0);
-    top: 0;
-  }
```

Comment:
It seems this is also covered in #12685, which was opened before this PR and has now been merged.